tests/pragmas/tbitsize.nim

```diff
@@ -0,0 +1,18 @@
+type
+  bits* = object
+    flag* {.bitsize: 1.}: cuint
+    opts* {.bitsize: 4.}: cint
+
+var
+  b: bits
+
```

Comment:
You should assert the sizeof() here to see it ended up in the generated C code. There is also a ccodeGenCheck but I forgot its syntax.